packages/rightdown/src/formatters/biome.ts

```
@@ -0,0 +1,257 @@
+import { Result, success, failure, makeError, type AppError } from '@outfitter/contracts';
```

ESLint

[error] Imports "Result" are only used as type.

(@typescript-eslint/consistent-type-imports)